config/crd/bases/instancemgr.keikoproj.io_instancegroups.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.9
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
Looks like you need to run `make controller-gen` followed by a `make manifests` - you are using an older version of controller-gen